apps/web/src/components/accessibility/accessibility-enhanced-components.tsx

```
@@ -0,0 +1,566 @@
+    <div className="space-y-2">
+      {label && (
+        <label 
+          htmlFor={inputId}
+          className={cn(
+            'text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70',
+            isSensitive && 'text-red-600 font-semibold',
+            isMedical && 'text-blue-600 font-semibold'
+          )}
+        >
+          {label}
+          {required && <span className="text-red-500 ml-1" aria-label="required">*</span>}
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)